src/meta/src/controller/catalog/drop_op.rs

```diff
@@ -239,11 +240,21 @@ impl CatalogController {
 
         // notify about them.
         self.notify_users_update(user_infos).await;
+        let dropped_tables = inner
+            .list_all_state_tables(Some(to_drop_state_table_ids.iter().copied().collect()))
```

Comment:
I may be missing something. `list_all_state_tables` selects `Table` relation. Since this happens after `Object::delete_many`, I think the corresponding rows will be deleted already and we cannot get any results here?

Reply:
You're right. Need to obtain them earlier.